opennms-webapp/src/main/java/org/opennms/web/filter/FilterUtil.java

```diff
@@ -50,4 +51,14 @@ public static String toFilterURL(List<String> filters) {
         }
         return( buffer.toString() );
     }
+
+    public static String[] parse(String filterString) {
+        String[] filterParameter = (filterString == null) ? null :Arrays.stream(filterString.split("&amp;"))
+                .filter(fp -> fp.startsWith("filter="))
+                .map(fp -> fp.replace("filter=", ""))
+                .map(fp -> fp = URLDecoder.decode(fp))
```

Comment:
decode(fp) seems to be deprecated
```suggestion
                .map(fp -> fp = URLDecoder.decode(fp, StandardCharsets.UTF_8.name()))
```

Reply:
removed deprecated method